bot/constants.py

```diff
@@ -89,6 +89,7 @@ class Emojis:
     christmas_tree = "\U0001F384"
     check = "\u2611"
     envelope = "\U0001F4E8"
+    pin = "\U0001f4cc"
```

Comment:
Is there a reason why you introduce an entirely new constant for this feature?

After invoking `.bm` now every message will have 2 emojis instead of one. Other user when reacting with `envelope` will not receive anything at all, and that is very confusing.

I do not think that we need to introduce a new constant and add a totally new emoji to do it, `envelope` will do just fine.

Reply:
As @sco1 said 
`I don't think it makes sense from a UX perspective to utilize the same reaction for multiple purposes when one of those purposes silently disappears after a timeout. This is already barely discoverable functionality, and with this implementation a user that does know of its existence doesn't know if it has timed out or if it's broken. A second emoji solves half of the problem, which might be sufficient.`

And i agree on this rather than using same emoji.

Reply:
If that's the case, you might want to think about how to avoid the confusion that is having 2 emojis, and one does not do anything at all, from the user's perspective.

If the envelope is purely for visual confirmation that the command worked, it might not be needed anymore since it is not adding anything else.

Reply:
I mean the envelope emoji doesn't get sent when it fails to dm . But pin still work and it give author an another chance to bookmark it. 

I like how it is now myself. 
An unknowing person will press both.
Or we can make it so it remove react when someone react with envelope.